crates/nu-cli/src/commands/last.rs

```diff
@@ -63,27 +63,21 @@ async fn last(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStr
     let (LastArgs { rows }, input) = args.process(&registry).await?;
     let v: Vec<_> = input.into_vec().await;
 
-    let rows_desired = if let Some(quantity) = rows {
-        *quantity
+    let end_rows_desired = if let Some(quantity) = rows {
+        *quantity as usize
     } else {
         1
     };
 
-    let count = rows_desired as usize;
-
-    let k = if count < v.len() {
-        v.len() - count
+    let beginning_rows_to_skip = if end_rows_desired < v.len() {
+        v.len() - end_rows_desired
     } else {
-        return Ok(futures::stream::iter(v).to_output_stream());
+        0
```

Comment:
The `else` here allows the table to print in its entirety if the value supplied to `last` is larger than the table.  Otherwise, the subtraction from v.len() and end_rows_desired will underflow and nothing will print out.